src/app.md

```diff
@@ -233,28 +233,35 @@ fn main() {
 <a name="ex-file-predicate"></a>
 ##  Recursively find all files with given predicate
 
-Use the file names and metadata to recursively find in the current directory JSON files modified within the last day.
-
 [![walkdir-badge]][walkdir] [![cat-filesystem-badge]][cat-filesystem]
 
+Use the file names and metadata to recursively find in the current directory 
```

Comment:
Please revise the sentence order. I would suggest "...JSON files modified within the last day, that are located in current directory"